skbeam/core/roi.py

```diff
@@ -757,3 +759,65 @@ def auto_find_center_rings(avg_img, sigma=1, no_rings=4, min_samples=3,
         edge_pts_xy = edge_pts_xy[~inliers]
 
     return center, image, radii
+
+def find_center_rings_hough(image, num_rings, rad_range, intensity_thresh, sigma, v_range = None):
+    """
+    Take angled cut or multiple incremented cuts of a image data.
```

Comment:
Do you mean the content or the extra "a"? I just got rid of the a

Reply:
@tacaswell is this resolved.  @madelinegullen thought she changed it so that the "a" was removed so it should now read `....cuts of image data`. I think she made all changes in the last item of this string of requests.  Is it on @madelinegullen to click resolve (or myself now) or you?  Unsure of best git practice.